volumina/widgets/exportHelper.py

```diff
@@ -35,63 +34,110 @@
 
 import logging
 
-logger = logging.getLogger(__name__)
 from volumina.utility import log_exception, PreferencesManager
+from volumina.layer import Layer
 
-###
-### lazyflow
-###
-_has_lazyflow = True
-try:
-    from lazyflow.graph import Graph
-    from lazyflow.request import Request
-    from lazyflow.utility import PathComponents
-except ImportError as e:
-    exceptStr = str(e)
-    _has_lazyflow = False
+import lazyflow
+from lazyflow.request import Request
+from lazyflow.utility import PathComponents
+from lazyflow.operators import OpMultiArrayStacker
+from lazyflow.operators.ioOperators import OpFormattedDataExport
 
 
-def get_settings_and_export_layer(layer, parent_widget=None):
-    """
-    Prompt the user for layer export settings, and perform the layer export.
+logger = logging.getLogger(__name__)
+
+
+def _get_export_slots(layer: Layer) -> typing.List[lazyflow.slot.Slot]:
+    """Returns export slots for layer
+
+    Args:
+        layer (Layer): Volumina Layer, its datasources will be checked for "dataSlot" member.
```

Comment:
```suggestion
        layer: Volumina Layer, its datasources will be checked for "dataSlot" member.
```
We already have type annotations.

Reply:
ahye, that's my autodoc plugin going mental - will remove :)